### PR TITLE
 Add new category - Retro 😎

### PR DESCRIPTION
**Pull request by BrunnerLivio**
## What type of PR is this? (check all applicable)

- [x] 🚀 Added Name
- [x] ✨ Feature
- [ ] ✅ Joined Community
- [ ] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Added a "Retro" section + Added GitHub users fnky and brunnerlivio to retro section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +156,11 @@
 - [Thomas George Thomas](https://github.com/Thomas-George-T/Thomas-George-T)
 - [Derek Nguyen](https://github.com/dereknguyen269/dereknguyen269)
 
+#### Retro 😎
+
```
Comment (abhisheknaiidu) [inline README.md:160]: remove this line space

File: README.md [hunk 2]
```diff
@@ -155,6 +156,11 @@
...
+- [Livio Brunner](https://github.com/BrunnerLivio)
+- [Christian Petersen](https://github.com/fnky)
```
Comment (abhisheknaiidu) [inline README.md:162]: change links to `https://github.com/BrunnerLivio/BrunnerLivio` from `https://github.com/BrunnerLivio`

Comment (BrunnerLivio) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (BrunnerLivio) [reply README.md]: Fixed

**Pull request merged**